src/commands/up.rs

```diff
@@ -90,6 +90,10 @@ pub struct UpCommand {
     /// All other args, to be passed through to the trigger
     #[clap(hide = true)]
     pub trigger_args: Vec<OsString>,
+
+    /// Only run a subset of the components
+    #[clap(long = "component", short = 'c')]
+    pub components: Vec<OsString>,
```

Comment:
This should probably be `String` rather than `OsString` - I don't think it's going to end up being passed back to the OS?.